Sources/Coordination/Flows/NavigationCoordinator.swift

```diff
@@ -17,11 +17,7 @@ public extension NavigationCoordinator where Self: ParentCoordinator {
     func openChildInline<T: NavigationCoordinator & ChildCoordinator>(_ childCoordinator: T) {
         root.delegate = childCoordinator
         openChild(childCoordinator)
-        
-        guard childCoordinator.root === root else {
-            assertionFailure("Child coordinators presented inline should have the same navigation controller as their parent.")
-            return
-        }
+        assert(childCoordinator.root === root, "Child coordinators presented inline should have the same navigation controller as their parent.")
```

Comment:
My understanding of `assert` over `assertionFailure` is that `assert` won't cause any runtime feedback if the condition does not evaluate to true?
Is this what we would want?
Currently the app would crash if `openChildInline` was called with a coordinator without the same root as the parent which feels right but interested to hear what the rationale is here.

Reply:
I think you have it right.  I was trying to be too efficient with coverage.  I will revert.